synapseAnnotations/data/cancer.json

```diff
@@ -158,29 +158,24 @@
     "maximumSize": 250,
     "enumValues": [
       {
-        "value": "Neurofibroma Cutaneous",
-        "description": "An intraneural or extraneural neoplasm arising from nerve tissues and neural sheaths of skin.",
-        "source": "https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0000622"
+        "value": "Cutaneous Neurofibroma",
+        "description": "A neurofibroma that grows along small branches of nerves in the dermis in patients with neurofibromatosis. It presents as a solid cutaneous tumor.",
+        "source": "http://purl.obolibrary.org/obo/NCIT_C128451"
       },
       {
-        "value": "Neurofibroma Subcutaneous",
```

Comment:
Do we have any data annotated with this key?

Reply:
No, I meant to delete it. According to Lu there is no such thing as subcutaneous.